CHANGELOG.md

```diff
@@ -1,9 +1,10 @@
 L-RdfToVirtuosoAndCkan
 ----------
 
-v1.1.6-SNAPSHOT
+v1.2.0-SNAPSHOT
```

Comment:
do we need to update version? isn't 1.1.5-SNAPSHOT enough to be reused?
If not we have to merge 1.1.5. changelog to 1.2.0 version